src/zAuction/index.ts

```diff
@@ -39,13 +43,16 @@ export const getSaleEventsFunction = (
   return async (domainId: string): Promise<DomainSaleEvent[]> => {
     const sales = await instance.listSales(domainId);
     const saleEvents = sales.map((e) => {
-      return {
+      const saleEvent: DomainSaleEvent = {
```

Comment:
I appreciate you changing this to the proper casting format (`const thing: Type = ` vs `as Type`)